cpp/core/compute/Runtime.h

```diff
@@ -49,34 +47,16 @@ struct SparkTaskInfo {
   }
 };
 
-/// ExecutionCtx is stateful and manager all kinds of native resources' lifecycle during execute a computation fragment.
```

Comment:
why remove comments?

Reply:
`Runtime` will no longer be about managing lifecycles of resources. It's more an C++ API for Spark-compatible query execution.

`objStore_` is still there to manage lifecycles but it will not be a long term solution. It might be removed once we developed a new way to ease Java-C++ object binding.